freqtrade/constants.py

```diff
@@ -512,6 +512,7 @@
                                 'minimum': 0,
                                 'maximum': 65535
                             },
+                            'secure': {'type': 'boolean', 'default': False},
```

Comment:
i'm not against this - but if we think about supporting this - we will have to also have some instructions on how to actually configure the server side.

We currently don't even mention a reverse proxy in the docs - but if the client side supports wss (which you can't get on the server without a reverse proxy) - we need to also show people how to configure that.

this can be as simple as an nginx snippet we know is working to serve both freqUi and websockets...